packages/cloudflare-workers-bindings-extension/src/show-bindings.ts

```diff
@@ -1,30 +1,57 @@
 import * as vscode from "vscode";
 import { importWrangler } from "./wrangler";
 
-type Config = ReturnType<
+export type Config = ReturnType<
 	ReturnType<typeof importWrangler>["experimental_readRawConfig"]
 >["rawConfig"];
 
-type Node =
-	| {
-			type: "env";
-			config: Config;
-			env: string | null;
-	  }
-	| {
+export type Environment = Required<Config>["env"][string];
+
+export type BindingType =
```

Comment:
What about tail consumer bindings?
Should we add anything to https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/docs/how-to/add-a-binding.md

I don't suppose we can share anything from wrangler without it being a massive pain...?

Reply:
[f984f60](https://github.com/cloudflare/workers-sdk/pull/7597/commits/f984f60563e8571c37d4ead3a606a14930ce184e)

Added logfwdr, pipelines & send_email. I left `data_blobs`, `text_blobs`, `wasm_modules` out from the implementation as they are legacy (as seen in the comment of the config type) and aren't available on enviornment.